### PR TITLE
13-label-factory : wrote tests and methods for LabelFactory

### LINKED ISSUE #13: Label Factory - Get All Labels (by IncidentTypeId)
### Feature Description

When creating a new incident the user will be prompted to  add a label. These labels are dependent upon the incident type. Within the form, the proper labels should be displayed to the user. 

1. write a test that queries the database based off of the incident type id
2. create a new file within the factory namespace that will be used to get factory labels. 
3. Inside of that new file create a class, within that class there should be a method that excepts the incidentTypeId param and then returns those labels within an Enumerable.  

### Related Issues/PRs

n/a

### PR DESCRIPTION
**Pull request by MattKraatz**
## Status
**READY**

## Testing

[X] I have added passing tests for all new functionality.

[X] I have confirmed that all test prior to this pull request still PASS.

## Documentation

[X] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
Build a Label Factory that returns a list of Labels according to a provided IncidentTypeId.

## Resolves Issue Number
Resolves #13 

## Deploy Notes

```sh
git pull --prune
git checkout 13-label-factory
dotnet test
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Make sure program builds and tests pass on your machine

## Impacted Files in Application
List general components of the application that this PR will affect:

* Factories/LabelFactory.cs
* LabelFactoryTests.cs

### CODE REVIEW

**Comment by daynewright** [comment]
LGTM

**Comment by garrett-vangilder** [comment]
All tests pass! LGTM

File: src/customer-service/Factories/LabelFactory.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+namespace customer_service
+{
+
+    /**
+     * Class: LabelFactory
+     * Purpose: Holds a singleton and can Get labels based on an IncidentTypeId
+     * Author: Matt Kraatz
+     * Methods:
+     *     List<Label> getLabels(int id) - Gets all Labels that match an IncidentType
```
Comment (matthamil) [inline src/customer-service/Factories/LabelFactory.cs]: `getLabels` should have an uppercase "G" like so: `GetLabels(int id)`

Apologies for being nitpicky :)

**Review by matthamil — APPROVED** [review]
LGTM

**Review by Anulfo — COMMENTED** [review]
LGTM

**Pull request merged**